src/internal/api/crypto/Null.hpp

```diff
@@ -243,10 +243,11 @@ class Blockchain final : public crypto::internal::Blockchain
     {
         return {};
     }
-    auto ProcessTransaction(
-        const Chain,
-        const opentxs::blockchain::block::bitcoin::Transaction&,
-        const PasswordPrompt&) const noexcept -> bool final
+    auto ProcessTransactions(
+        const Chain chain,
```

Comment:
Might be a good idea to remove the unused argument names and save time and effort when we remove the suppression of the relevant warning.

Reply:
MT-84

Reply:
Just doing it would probably be easier than adding TODO notes here.